addons/vectors/stringtable.xml

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Which of these is right?

Reply:
It seems Tabler saves _stringtable.xml_ files without [BOM](https://en.wikipedia.org/wiki/Byte_order_mark). On your master branch, only vectors/ , versioning/ and xeh/ have BOM. All the others do not. I would assume that the one without BOM (my branch) are right.

Reply:
I already mentioned somewhere that these 3 had invalid bom in the past. We also had a couple script files with invalid I think.

Reply:
On master branch backpacks module stringtable still has bom.